src/docs/spec.md

```diff
@@ -438,17 +448,19 @@ Illegal case 3:
 Example ([download](https://raw.githubusercontent.com/mapping-commons/sssom/master/examples/embedded/mp-hp-exact-0.0.1.sssom.tsv)):
 
 ```
-#creator_id: "https://orcid.org/0000-0002-7356-1779"
 #curie_map: 
 #  HP: "http://purl.obolibrary.org/obo/HP_"
 #  MP: "http://purl.obolibrary.org/obo/MP_"
-#  skos: "http://www.w3.org/2004/02/skos/core"
+#  skos: "http://www.w3.org/2004/02/skos/core#"
+#  orcid: "https://orcid.org/"
+#creator_id:
```

Comment:
Thanks for spotting / fixing this bug; the `EntityReference` on `creator_id` needing to be a `curie`. This tripped me up recently as well.